src/test/java/org/opensearch/ad/ml/EntityColdStarterTests.java

```diff
@@ -842,11 +843,56 @@ public int searchInsert(long[] timestamps, long target) {
     }
 
     public void testAccuracyTenMinuteInterval() throws Exception {
-        accuracyTemplate(10);
+        accuracyTemplate(10, 0.5f, 0.5f);
     }
 
     public void testAccuracyThirteenMinuteInterval() throws Exception {
-        accuracyTemplate(13);
+        accuracyTemplate(13, 0.5f, 0.5f);
+    }
+
+    public void testAccuracyOneMinuteIntervalNoInterpolation() throws Exception {
+        // make sure AnomalyDetectorSettings.ALLOW_INTERPLATION_IN_COLDSTART is set to false now
+        assertEquals(false, AnomalyDetectorSettings.ALLOW_INTERPLATION_IN_COLDSTART);
```

Comment:
love showing the improvement! Should we disable interpolation for other tests or test for other interval lengths?

Reply:
The improvement for other tests are not that much since the interpolated data for those cases are much less than the one-minute case. Thus, the improvement is not that obvious.

I kept other tests unchanged so cover the path using interpolation. Otherwise, our coverage will drop.